### ISSUE TITLE
[Request] UI Design

### ISSUE THREAD

**Issue by ThinkDigitalSoftware** [thielepaul/photoprism-mobile#89]
Could we get an agreed-upon UI design?

**Comment by ippocratis** [comment]
chromium based browsers can create shortcuts as webapps in full screen and dope looking .
give a ip:port url and create a shortcut.
the mobile app is so barebones atm

**Comment by ThinkDigitalSoftware** [comment]
Yeah, we need a design

**Comment by thielepaul** [comment]
not sure what this is about, are you referring to the fact that app and webapp look differently?

**Comment by ThinkDigitalSoftware** [comment]
> not sure what this is about, are you referring to the fact that app and webapp look differently?

Yes. And that it has a nice design. We should have a UI designer come up with one so someone can build it and we can be a little consistent with the design. Idk. Just thinking ahead

**Comment by thielepaul** [comment]
sure, that would be nice. I will focus on functionality for now. There is already more to be done there than I manage to do. But if someone wants to do this, I am happy to accept PRs.

**Comment by ThinkDigitalSoftware** [comment]
Sure thing! Thanks for the work this far.
